Command/ListCommand.php

```diff
@@ -75,10 +76,18 @@ protected function configure(): void
     protected function execute(InputInterface $input, OutputInterface $output)
     {
         $table = new Table($output);
-        $table->setHeaders(['Cron job', 'Description']);
+        $table->setHeaders(['Cron job', 'Description','Next Run']);
         foreach ($this->cronJobManager->getCronJobs() as $cronJob) {
-            $table->addRow([get_class($cronJob), $cronJob->getDescription()]);
+
+            if (CronExpression::isValidExpression($cronJob->getExpression())) {
+                $cronExpression = CronExpression::factory($cronJob->getExpression());
+                $nextRunDate = $cronExpression->getNextRunDate()->format('Y-m-d H:i:s');
+            }
+
+            $table->addRow([get_class($cronJob), $cronJob->getDescription(), $nextRunDate]);
```

Comment:
`$nextRunDate` may not exist if the previous condition is false. add a default value